toolkits/springernature/packages/springernature-forms/scss/30-mixins/_forms.scss

```diff
@@ -0,0 +1,10 @@
+@mixin forms--input {
```

Comment:
This mixin is only used in a single file, so can be simplified by including it directly at the top of that file